packages/editor-ui/src/components/mixins/pushConnection.ts

```diff
@@ -246,14 +251,6 @@ export const pushConnection = mixins(
 							message: `${action} <a href="https://docs.n8n.io/nodes/n8n-nodes-base.wait/" target="_blank">More info</a>`,
 							type: 'success',
 							duration: 0,
-							onLinkClick: async (e: HTMLLinkElement) => {
```

Comment:
can you delete the code that handles `onLinkClick`? Since this is now obsolete

Reply:
Done. I didn't realize it's not used anywhere else